source/docs/magento2/add-new-attribute.md

```diff
@@ -0,0 +1,187 @@
+- modules: ["./src"],
++ modules: ["./mymodule", "./src"],
+  serverModules: [
+    { name: "FrontCommerce", path: "server/modules/front-commerce" },
+-   { name: "Magento2", path: "server/modules/magento2" }
++   { name: "Magento2", path: "server/modules/magento2" },
++   { name: "Acme", path: "./mymodule/server/modules/acme" }
+  ]
+};
+```
+
+### Expand the Product definition
+
+Create a file named `schema.gql` and type in the following code. It means that we are expanding the definition of a product: it can now be rated.
+
+```js
+// mymodule/server/modules/acme/schema.gql
+extend type Product {
+  rate: Float
+}
+```
+
+### Implement the resolver
+
+We must now define what Front-Commerce should fetch when `rate` is requested in a GraphQL query; in other words we must write the code that will **resolve** the queries for the rates.
+
+Create a file named `resolvers.js` and type in the following code. It implements the resolver.
+
+```js
+// mymodule/server/modules/acme/resolvers.js
+export default {
+  Product: {
+    rate: parent => {
+      return parseFloat(parent.rate);
+    }
+  }
+};
+```
+
+### Declare the module
+
+If we want our code to be taken into account when the module is loaded, we must reference it.
+
+In the file named `index.js`, type in the following code. It references the schema and the resolver from earlier.
+
+```js
+// mymodule/server/modules/acme/index.js
+import typeDefs from "./schema.gql";
+import resolvers from "./resolvers";
+
+export default {
+  namespace: "Acme",
+  typeDefs: typeDefs,
+  resolvers: resolvers
+};
+```
+
+## Discover the playground
+
+By typing `yourhostname/playground` or `yourhostname/graphiql` (in our case `localhost:4000/playground`) in your browser address bar, you can access a GraphQL playground with a nice GUI to test your queries. For instance, type the following code in the left pane and press `Ctrl` + `Enter` (or click the arrow button)
+
+```graphql
+{
+  product(sku: "yourSKUhere") {
+    name
+    rate
+  }
+}
+```
+
+From this GraphQL query, you should get a JSON content that looks like this:
+
+```json
+{
+  "data": {
+    "product": {
+      "name": "Your product name",
+      "rate": // the value stored in your backoffice (Magento 2 in this case)
+    }
+  }
+}
+```
+
+### Overriding the files
+
+In **this** example, the files to be overridden are `ProductSynthesisFragment.gql` and `Synthesis.js` both located at `node_modules/front-commerce/src/web/theme/modules/ProductView/Synthesis/`. Copy and paste them into `mymodule/web/theme/modules/ProductView/Synthesis`.
+
+#### How to find which files to edit
+
+On your web page, right click on the element you wish to modify, then click on **Inspect Element**. In the inspector, you can look for the `className` that holds the elements you want to edit. Then you can search for that keyword in your `node_modules/front-commerce/src` folder and you will find the right files.
+
+We've included images to illustrate our example.
```

Comment:
Rather than saying everything in one paragraph and then adding images, I'd suggest to split the paragraph and put the images exactly when you are talking about them: 1. part about the inspector, 2. part about the search in your files.